EIPS/eip-6093.md

```diff
@@ -0,0 +1,371 @@
+---
+eip: 6093
+title: Custom errors for commonly-used tokens
+description: Lists custom errors for common token implementations
```

Comment:
I think we can expand the description a bit. Maybe:

```suggestion
description: Structured custom errors for common tokens (EIP-20, EIP-721, and EIP-1155)
```

Reply:
Thanks for the suggestion. I'm slightly more biased toward your proposal, but there's a [competing comment](https://github.com/ethereum/EIPs/pull/6093#discussion_r1042115588) I've already accepted.

I'll leave it open in case someone else wants to comment on this.